Headers/DebugServer2/Target/Linux/Thread.h

```diff
@@ -31,6 +31,9 @@ class Thread : public ds2::Target::POSIX::Thread {
 #endif
 
 protected:
+  void fillStoppointData();
```

Comment:
Also, this function name has a lifespan of 2 or 3 commits anyway. I still think it's the correct name though, based on what the function does.